tests/service/backend/test_mongodb.py

```diff
@@ -93,15 +93,15 @@ def test_create(mongo_backend: GetMongoBackend, parameterized_entity: Parameteri
     backend = mongo_backend("write")
 
     # Create a single entity
-    entity_from_backend = backend.create([parameterized_entity.backend_entity])
+    entity_from_backend = backend.create([parameterized_entity.parsed_entity])
 
     assert isinstance(entity_from_backend, dict)
-    assert entity_from_backend == parameterized_entity.backend_entity
+    assert entity_from_backend == parameterized_entity.parsed_entity
 
     # Create multiple entities
     raw_entities = [
         {
-            "uri": "http://onto-ns.com/meta/1.0/Test",
+            "identity": "http://onto-ns.com/meta/1.0/Test",
```

Comment:
Try adding an identity which is not a URI

Reply:
As a test, you mean? Sure.